Project-Lynx/src/main/java/lynx/commands/utilities/Truncate.java

```diff
@@ -0,0 +1,9 @@
+package lynx.commands.utilities;
```

Comment:
I don't agree this should be under `lynx.commands.utilities`

`lynx.commands` is for the `Command` derived classes.

Please consider putting it under `lynx.misc`